direct/core/src/main/java/cz/o2/proxima/direct/core/OnlineAttributeWriter.java

```diff
@@ -67,4 +67,12 @@ default void rollback() {
   @SuppressWarnings("unchecked")
   @Override
   Factory<? extends OnlineAttributeWriter> asFactory();
+
+  default boolean isTransactional() {
+    return false;
+  }
+
+  default TransactionalOnlineAttributeWriter toTransactional() {
```

Comment:
maybe we should rename this to `asTransactional`, to make things more consistent?

Reply:
Sounds good.

Reply:
Hm, maybe the most consistent would be `transactional()`, because we already have methods `online()` and `bulk()` which do pretty much the same.